### ISSUE TITLE
LDAP / Active Directory Integration

### ISSUE THREAD

**Issue by walalm** [alextselegidis/easyappointments#128]
Wouldn't be great to map your users and login with LDAP / Active Directory? Please add that functionality.

**Comment by alextselegidis** [comment]
I'm not sure if I really understand what you need ... Could you please clarify your question?

**Comment by walalm** [comment]
Integrate the application with Active Directory / LDAP so the users to login use that method.

**Comment by alextselegidis** [comment]
I don't think this could be a priority right so it won't be implemented at least for the upcoming releases. Thank you for taking your time to post this ticket.

**Comment by alextselegidis** [comment]
Re-openning the ticket as a future enhancement.

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by croftj01** [comment]
Hello,
Ditto here, LDAP would be a great addon.

**Comment by achouvardas** [comment]
Any update regarding the CAS / LDAP authentication feature ?

**Comment by h-e-l-l-o-w-o-r-l-d** [comment]
Really looking forward to this in 1.5! <3

**Comment by alextselegidis** [comment]
Almost there! :)

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by aparrot89** [comment]
Hi.
Would it be possible to implement the possibility of connecting to easyappointment with a SSO ?
With oidc for example?

**Comment by alextselegidis** [comment]
Hello! 

That is in general possible although for this integration we'll be syncing users out of the LDAP directly and into Easy!Appointments

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by compumatter** [comment]
Hello, 

I have just installed the latest 1.5.0-4 because I had seen the blog info that LDAP would be part of it.  I do not see where to configure it.  Are there any docs on this to refer to?

Thanks.

Jay

**Comment by alextselegidis** [comment]
Good news! LDAP is now completed and pushed to develop. 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>
